docroot/themes/custom/uids_base/scss/global.scss

```diff
@@ -441,6 +446,32 @@ body:not(.page-node-type-article, .page-node-type-page).layout-builder-disabled
   }
 }
 
+
+
+// Section w/ color background padding
+
+div[class*="uids-component--"] {
+  .lb__container {
+    padding-top: $mobile-width-gutter;
+    padding-bottom: $mobile-width-gutter;
+
+    @include breakpoint(sm) {
+      padding-top: $desktop-width-gutter;
+      padding-bottom: $desktop-width-gutter;
+    }
+  }
+
+  &.section-padding__removed {
```

Comment:
I added a new `section-padding__removed` layout builder style to remove the default padding for sections with a background color.  There are two sections that will need this option on the home page, the stats section and the events section.